src/StereoDB/StereoDb.fs

```diff
@@ -122,8 +130,44 @@ type internal StereoDb<'TSchema when 'TSchema :> IDbSchema>(logger: ILogger, sch
             with
                 ex -> ()
     }
+
+    let createHeartbeatTopic = task {
+        use adminClient = AdminClientBuilder(AdminClientConfig()).Build()
+        try
+            do! adminClient.CreateTopicsAsync(
```

Comment:
TODO @oleksiikraievyi add handling if topic exists + reasonable error messages 
also discuss if we should retry or fail db spinning up